janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/formats/util/JanusGraphVertexDeserializer.java

```diff
@@ -77,7 +74,7 @@ private boolean edgeExists(Vertex vertex, RelationType type, RelationCache possi
 
     // Read a single row from the edgestore and create a TinkerVertex corresponding to the row
```

Comment:
(nitpick) Please also update this comment as it explicitly mentions that a `TinkerVertex` will be created:

```suggestion
    // Read a single row from the edgestore and create a Vertex corresponding to the row
```

Reply:
Ack